src/hotspot/share/opto/vectornode.cpp

```diff
@@ -920,6 +924,50 @@ bool VectorNode::is_vector_bitwise_not_pattern(Node* n) {
   return false;
 }
 
+bool VectorNode::is_scalar_unary_op_with_equal_input_and_output_types(int opc) {
+  switch (opc) {
+  case Op_SqrtF:
```

Comment:
Our style guide does not really says anything about indentation of switches. I would have suggested, though, to indent it:
```
switch (opc) {
  case Op_SqrtF:
  ....
}
```

Reply:
The rest of the file does it the way I have it, so I think I'll keep it.